apis/sgv2-quarkus-common/src/test/resources/application.yaml

```diff
@@ -11,6 +11,15 @@ quarkus:
         host: localhost
         port: 8091
 
+"%test":
```

Comment:
Is this needed when `application.yaml` itself is under `src/test`? (shouldn't be included in non-test profiles)

Reply:
Yes., Here %test represent the test profile of quarkus which is used when running the unit test.